### DISCUSSION TITLE
Cannot send broadcasted UDP messages after win10 update (2004)

### DISCUSSION THREAD

**Discussion by misfit22** [dannagle/PacketSender#222] [General]
Hello,

maybe someone else also experienced this, i just cannot send broadcasted udp packets from any of my applications. Happened exactly after a bulky windows update (from v1909 to v2004). First thing was trying to send packages using packetsender and sniff the network with wireshark but no packages reach that level to monitor. I updated the ethernet interface driver, checked the routing tables, tried to run with administrator but nothing helped.

Thank you!

**Comment by misfit22** [comment]
Ok problem solved, windows 10 firewall was blocking udp traffic.

**Reply by dannagle** [reply]
Glad you found the answer. I've seen more complaints about this recently. This particular Windows update must have a more aggressive firewall. I'm leaving this ticket open as a reminder to update the docs or put a warning in the app itself.
